src/qt/locale/bitcoin_nl.ts

```diff
@@ -1736,13 +1736,17 @@ p, li { white-space: pre-wrap; }
         <source>BIP70 payment requests are deprecated and disabled by default. Restart with -enable-bip70 if you absolutely have to use this functionality.
 
 Use this functionality with extreme caution.</source>
-        <translation type="unfinished"></translation>
+        <translation>BIP70 betaal verzoeken zijn verouderd en daarmee standaard uitgeschakel. Herstart BIP70 betaalverzoeken met -enable-bip70 als je absoluut zeker bent dat je deze functionaliteit wil gebruiken.
+
+Gebruik deze functie met extreme voorzichtigheid.</translation>
```

Comment:
"uitgeschakel" -> "uitgeschakeld"

> Herstart BIP70 betaalverzoeken met -enable-bip70

gives incorrect meaning, because what needs to be restarted is `dogecoin-qt`, so replace "BIP70 betaalverzoeken" here with "de applicatie"

Reply:
Good suggestions, i have changed the translation